azure-pipelines.yml

```diff
@@ -172,3 +180,25 @@ steps:
       containerRegistry: "azurecr-oaf"
       tags: |
         $(IMAGE_TAGS)
+  
+  - task: Bash@3
+    displayName: Use latest image tag
+    inputs:
+      targetType: "inline"
+      script: |
+        sed -i 's/tag:.*/tag: $(IMAGE_TAGS)/' values.yaml
+
+  - task: CopyFiles@2
+    displayName: Copying source files
+    inputs:
+      SourceFolder: "./"
+      Contents: "*.yaml"
+      TargetFolder: "$(Build.ArtifactStagingDirectory)"
+
+  - task: PublishBuildArtifacts@1
```

Comment:
hey @amydoxym aren't we pushing already [here](https://github.com/one-acre-fund/fineract-frontend-v2/blob/9496bace2fc8b8fb76f0a4dd00de928a9df238bb/azure-pipelines.yml#L177)

Reply:
@shejasabin Yes, we are. This step is  to publish all the Azure build package. The step you mention is to builld and push the image on the registry